src/clue.js

```diff
@@ -117,10 +117,29 @@ var Patio = {name: 'Patio'}
 
 
 // Characters Collection
-var charactersArray = [drOrchid, drOrchid, profPlum, missScarlet, mrsPeacock, mrMustard];
+var charactersArray = ['mrGreen', 'drOrchid', 'profPlum', 'missScarlet', 'mrsPeacock', 'mrMustard'];
 
 // Rooms' Collection
-var roomsArray = [rope, knife, candlestick, dumbbell, poison, axe, bat, trophy, pistol];
+var weaponsArray = ['rope', 'knife', 'candlestick', 'dumbbell', 'poison', 'axe', 'bat', 'trophy', 'pistol'];
 
 // Weapons Collection
-var weaponsArray = [DinningRoom, Conservatory, Kitchen, Study, Library, BilliardRoom, Lounge, Ballroom, Hall, Spa,  LivingRoom, Observatory, Theater, GuestHouse, Patio];
+var roomsArray = ['DinningRoom', 'Conservatory', 'Kitchen', 'Study', 'Library', 'BilliardRoom', 'Lounge', 'Ballroom', 'Hall', 'Spa',  'LivingRoom', 'Observatory', 'Theater', 'GuestHouse', 'Patio'];
+
+function randomSelector(array){
+  return array[(Math.round((Math.random()) * array.length-1))];
```

Comment:
Math.round() will sometimes round up and sometimes round down. To pass the test you would need to use Math.floor() and multiply it by array.length or Math.ceil() and multiply it by array.length - 1.